fedimint-server/src/net/api.rs

```diff
@@ -547,6 +553,15 @@ pub fn server_endpoints() -> Vec<ApiEndpoint<ConsensusApi>> {
                 Ok((&fedimint.session_status(index).await).into())
             }
         },
+        api_endpoint! {
+            SHUTDOWN_ENDPOINT,
+            ApiVersion::new(0, 0),
```

Comment:
This ideally should be a version the call was introduced in, and I think we're at `0.2`, and should increment to 0.3. https://github.com/fedimint/fedimint/blob/bffe4660774db857ab5672f654e6eae96522246b/fedimint-server/src/config/mod.rs#L211

Reply:
Thought being a quick EoL fix for fedimint 0.3 branch, it all probably doesn't matter.

Reply:
If we update here, should we also update master?

Reply:
I think so, yes. For consistency at least. I doubt it makes a difference in practice, and mostly pointing it out so in the future we don't forget.

Reply:
Updating master in https://github.com/fedimint/fedimint/pull/5640